### DISCUSSION TITLE
Sort functions for shop items

### DISCUSSION THREAD

**Discussion by MeganJClark** [gameserverapp/Platform#206] [Suggestions]
My shop is growing, and becoming hard to navigate. A couple of sort functions would be helpful - alphabetical and by label. The case - I started adding blueprints to the shop two days ago, got busy with RL work, and I'm sitting down now to finish adding the items, but I am not sure where I left off on the list of items I'm adding. If I could sort them according to the label, I could quickly see all of the blueprints I have added already.

**Comment by elusive-ttv** [comment]
This would be an incredible feature! We have hundreds of items in our shop and it's near impossible to sort through and find a specific one sometimes. Alphabetical sorting would solve that problem for us entirely!

**Comment by mvaessen** [comment]
Hi both,

Thanks for your suggestion!
For my understanding, are you referring to a sorting option on the community website side, or on the dashboard?

Reason i'm asking is due to the sorting order on the dashboard being used to determine the sorting on the community website.

**Reply by MeganJClark** [reply]
I mean the dashboard side. The sort and search functions in the game and on the website are good.

**Comment by elusive-ttv** [comment]
I am speaking of the dashboard.
